cypress/integration/bookings/sort.js

```diff
@@ -0,0 +1,73 @@
+import { checkBookingsOrder } from '../../lib/helper.js'
+
+describe('Sort bookings', () => {
+  it('Should be sorted by id by default', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/booking/list')
+
+    cy.contains("Booking 1")
+    cy.contains("Booking 10")
+
+    //check if the bookings are sorted by id the first time, taking all of them 
+    //and checking their order, which should be ascending
+    checkBookingsOrder(1, 1)
+    //the first parameter represents the first booking and the second parameter 
+    //represents the number that is added to obtain the next booking
+  })
+
+  it('Check sorting by booking id', () => {
+    cy.login('normal@example.com', 'password')
+    cy.visit('http://localhost:3000/booking/list')
+
+    cy.contains("Booking 1")
+    cy.contains("Booking 10")
+    cy.contains("Booking 11").should('not.exist')
+
+    //should be sorted ascending by id (ascending by default)
+    cy.get('[id="sort-by"]').first().click()
+    cy.get('[id="sort-by-id"]').first().click()
+    checkBookingsOrder(1, 1)
```

Comment:
shouldn't the second parameter be an array? I find the way the function params confusing

Reply:
no, it shouldn't be array. I explained above what each parameter represents. if I don't use these parameters I will have to write repetitive code.
practically, the first parameter represents the number of the first booking, and the second represents the number that I collect to obtain the next parameter. If we start with booking 1 and add 1, I will get 2, then 3 and so on. If we start with booking 20 and add -1, we will get 19, then 18 and so on

Reply:
Oh, got it. Let's rewrite this function to take arrays as parameters because in the future we'll have more filtering options and we want this function to be as general as possible